src/main/resources/application.properties

```diff
@@ -160,20 +160,20 @@ logging.logback.rollingpolicy.max-file-size=20KB
 logging.logback.rollingpolicy.max-history=1
 
 ### SMS Gateway URL details
-sms-username=<Enter SMS username>
-sms-password=<Enter SMS password>
-send-message-url=<Enter SMS sending URL here>
+sms-username=amrit
+sms-password=Amrit@321
+send-message-url=api/mt/SendSMS?
 source-address=AIDSHL
 sms-message-type=SERVICE_EXPLICIT
 sms-entityid=1201161708885589464
 
  
 
 ### Email Gateway URL details
-email-gateway-url=<Enter email gateway URL here>
-send-email-url=<Enter email sending URL here>
-email-username=<Enter AMRIT email username>
-email-password=<Enter AMRIT email password>
+email-gateway-url=https://api.unicel.in
+send-email-url=SendSMS/sendmsg.php?
+email-username=AMRIT
+email-password=Psmri@12
```

Comment:
@swetatech24 Replace it with the placeholder. Do not put credentials in properties file.